extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -104,6 +104,10 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 			slotTargetingData.uap_c = uapContext.getCreativeId();
 			slotTargetingData.outstream = slotTargeting.getOutstreamData() || 'none';
 
+			if (['TOP_LEADERBOARD', 'BOTTOM_LEADERBOARD'].includes(slot.name)) {
```

Comment:
Function `includes` is ES6 method so it won't work on IE. Please replace it with:
`['TOP_LEADERBOARD', 'BOTTOM_LEADERBOARD'].indexOf(slot.name') !== -1`